services/rook-ceph-cluster/1.10.8/defaults/cm.yaml

```diff
@@ -257,7 +257,7 @@ data:
         enabled: true
         bucketName: dkp-loki
         storageClassName: dkp-object-store
-        enableOBCHealthCheck: true
```

Comment:
why is this being flipped ?

Reply:
This enables the job that waits for the OBC to be ready then reconciles the HR. 

We don’t really need this anymore since we don’t expect the Loki HR to fail. But as a safety net, I think I might keep it enabled (it won’t do any harm from what I can tell)